### DISCUSSION TITLE
Cannot find 3.7.0-centos image in Docker hub

### DISCUSSION THREAD

**Discussion by heichong** [apache/apisix#10643] [General]
我在docker hub中找不到3.7.0-centos，只找到了以下两种：
![image](https://github.com/apache/apisix/assets/1830250/5dbd0e63-2939-4db8-a15f-5e5595d51aba)

请问从哪里能找到3.7.0-centos

**Comment by kayx23** [comment]
It was removed: https://github.com/apache/apisix-docker/pull/512

**Reply by heichong** [reply]
thanks for reply
